lib/geocoder/googlegeocoder.js

```diff
@@ -270,8 +270,12 @@ GoogleGeocoder.prototype._reverse = function (query, callback) {
     params.language = query.language;
   }
 
-  if (query.region) {
```

Comment:
@nchaulet because according to the docs it is not being used by google for reverse geocoding. It is only used for direct geocoding. I believe `place_id` or `latlng` already contain distinct information about the exact spot on the Earth's surface which you want to get info about, you can't influence this anyhow by supplying `region`.